src/Files.App/Views/LayoutModes/ColumnViewBase.xaml

```diff
@@ -147,6 +147,7 @@
 			Visibility="Collapsed" />
 
 		<SemanticZoom
+            x:Name="RootGridZoom"
```

Comment:
Is spacing off? Might want to change your Visual Studio settings to use tabs.

Reply:
I'm sorry it seems to reset each time I open it. I'll reinstall it

Reply:
![Tabs](https://user-images.githubusercontent.com/102259289/190727932-7f08c4c6-0056-4c82-b149-762ee5ab842a.png)
Are these the right settings?

Reply:
Yes